tests/testthat/test_survival_surv_parametric.R

```diff
@@ -144,3 +144,50 @@ test_that("loglogistic", {
   expect_equal(unname(as.matrix(p$distr$cdf(data = t(cdf)))),
     matrix(c(rep(0.2, 50), rep(0.8, 50)), byrow = TRUE, nrow = 2))
 })
+
+task = tsk("rats")
+task$filter(sample(task$nrow, 100))
+
+test_that("discrete - aft", {
+  p_cont = lrn("surv.parametric", discrete = FALSE)$train(task)$predict(task)
```

Comment:
if you change here to eg `predict(task, sample(task$row_ids, 10))` you will notice differences